tests/karate/src/test/resources/karate-config.js

```
@@ -7,9 +7,9 @@ function fn() {
 
   // Basic configuration
   var config = {
-    baseUrl: "http://graphql-engine-test:8080/v1/graphql",
+    baseUrl: karate.properties['baseUrl'] || 'http://graphql-engine-test:8080/v1/graphql',
     webhookUrl: "http://webhook:3001",
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)